src/lib/plugin.js

```diff
@@ -20,6 +20,8 @@ const util = require('util')
 const backoffMin = 1000
 const backoffMax = 30000
 const defaultConnectTimeout = 60000
+const wsReconnectDelayMin = 1
+const wsReconnectDelayMax = 300
```

Comment:
If the plugin is disconnected for any substantive amount of time it can cause a connector to lose money